src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -283,14 +283,19 @@ class WorkspaceMembersPage extends React.Component {
     }
 
     render() {
-        const policyMemberList = _.keys(lodashGet(this.props, 'policyMemberList', {}));
-        const removableMembers = _.without(policyMemberList, this.props.session.email, this.props.policy.owner);
-        const data = _.chain(policyMemberList)
-            .map(email => this.props.personalDetails[email])
-            .filter()
-            .sortBy(person => person.displayName.toLowerCase())
-            .map(person => ({...person})) // TODO: here we will add the pendingAction and errors prop
-            .value();
+        const policyMemberList = lodashGet(this.props, 'policyMemberList', {});
+        const removableMembers = [];
+        let data = _.map(policyMemberList, (policyMember, email) => {
```

Comment:
This logic is a bit weird. You are using a map to build data but also using it to generate removableMembers values. I feel like this would have been easier with a `_.each` to do different things.

Anyway besides that, I feel like data would be a lot cleaner with the `chain`. Its weird that it's doing some things here and then in the next line it does sortBy. Also its not doing filter so it can end up with empty values which will fail when lowerCase is done,
![image](https://user-images.githubusercontent.com/14356145/190282307-5dfc22bf-3b07-49e9-ae2d-36f17192a1bc.png)

I think data should be generated via chain and removableMembers should keep its operations separate. Or use a _.each to build both but that's extra work which I feel is not worth it just yet here.

Reply:
Updated to use each